src/content/release-notes.mdx

```diff
@@ -17,6 +17,7 @@ id: release-notes
 - Fix network policies configuration to limit buildkit access to the nginx controller  <!-- 7505 -->
 - Add Okteto Insights Token to the Admin UI  <!-- 7473 -->
 - Remove customer data from mixpanel events  <!-- 7390 -->
+- Update structure of the Okteto Documentaion to make content easier to navigate. All changes [here](https://github.com/okteto/docs/pulls?q=is%3Apr+label%3Adoc-restructure+)
```

Comment:
@omnomagonz Added a small note about structure changes to the release notes.

Reply:
I can see why we would want to add this here, although I'm not entirely sure it belongs to the release notes (as opposed to a blog post for eg)

Note: There's a typo in `Documentaion`

Reply:
It's just that this is not strictly part of the release. I mean, the docs are for the Chart, the CLI, Cloud, manifest, samples, etc so the scope of the change is larger in principle than the 1.16 release.
Not totally against it but I would like to get more input here since we haven't done this before afaik.

Reply:
@maroshii although it's not ideal, I think it's a good place to mention it because we bump the doc versions with each chart release.
@codyjlandstrom but I think the comment is insufficient. At a minimum, I would mention the new location for "release notes" and the Chart values reference, which are commonly accessed by our customers. And a blog post would be ideal, but not sure if we have capacity to do it

Reply:
@pchico83 I like those suggestions/callouts 👍 

Cody and I can draft a blog post but I suspect it'll be a January publish date given upcoming holidays/PTO

Reply:
@omnomagonz I think the video/text message is fine. My 2$ is that a blogpost isn't worth the investment

Reply:
That sounds _extra_ good to me then 😄